docs/manual/dynamic_specs.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: rpm.org - Package Build Process
+---
+# Dynamic Spec Generation
+
+Since rpm 4.19 RPM supports parsing dynamically generated specs. This
+allows the build scripts (**%build** or **%install**) to create parts
+of the spec file. The parts are read in and parsed after **%install**
+and before **%check**. Because of this they obviously can't contain
+the build scripts but are intended to create sub packages based on the
+build results.
+
+The files need to be placed in the **%{specpartsdir}** (also available
+as **$RPM_SPECPARTS_DIR** in the build scripts) and have a
+**.specpart** postfix. The directory is created by **%setup** in the
+**buildsubdir**. Scripts must fail if it is not present. They should
```

Comment:
This sentence sounds contradictory to what follows it, i.e. that they in fact don't have to fail if they do some kind of fallback.

Reply:
Reworded. There are just a little to many ifs and whens to make this nice to read literature. Or my poetic qualities are just lacking...